### ISSUE TITLE
lib.externals - lodash 

### ISSUE THREAD

**Issue by joshpitkin** [dherges/ng-packagr#265]
## Type of Issue

```
[ ] Bug Report
[ ] Feature Request
[*] I'm stuck
```

## Description

BUILD ERROR
'forEach' is not exported by node_modules\lodash\index.js
Error: 'forEach' is not exported by node_modules\lodash\index.js
....

#### How To Reproduce

in a component.ts use the following to import:

```
import * as _ from 'lodash';

  ngOnInit() {
    const x = [];
    _.forEach(x, (row: any) => {
      console.log(row);
    });
}
```

#### Expected Behaviour

in package.json some like this should work?

```
  "ngPackage": {
    "lib": {
      "entryFile": "public_api.ts",
      "externals": {
        "_": "lodash"
      }
    }
  }

```

#### Version Information

```
ng-packagr: v1.5.0
node: v6.9.3
@angular: v4.4.4
```

I can't figure out how to setup the lib.externals for this..  tried lots of variations no luck..
Studied #129 , no example seems to help for this case.

**Comment by dherges** [comment]
Ah shit. These externals need improvement.

Your confit should be:

“lodash”: “something”

Lodash because that is the import path in the typescript statement. Something needs to be replaced with the UMD module id. The only reliable way if figuring it out is looking at  dist JavaScript code. Suspicion is that it’s maybe “_” the underscore for lodash because you’re using lodash as “_.foo()” when it’s registered on the global scope.

Cheers, David

**Comment by joshpitkin** [comment]
So this is strange.  I discovered the lodash error was actually coming from a lodash import within another library, not my own.  The error was quite misleading as adding 
  "lodash": "_" 
(and a variety of other combinations) to externals did nothing to it..

When I finally found the culprit library and added it to the externals it everything cleared up.

Yeah, looking forward to something better for externals if you figure it out.. 

Thanks for the quick reply!
